bin/cli.js

```diff
@@ -87,14 +35,22 @@ yargs
       type: 'integer',
       alias: 'p'
     }
-  }, dev)
+  }, (argv) => {
+    commands.dev(argv).catch((err) => {
+      helpers.showErrorAndQuit(err);
+    });
+  })
   .command('deploy', 'Pushes compiled assets to gh-pages branch on remote', {
     prefix: {
       describe: 'Prefixes all links with supplied string',
       type: 'string',
       defualt: null
     }
-  }, deploy)
+  }, (argv) => {
+    commands.deploy(argv).catch((err) => {
+      helpers.showErrorAndQuit(err);
+    });
```

Comment:
Nit: You can actually simplify these a little to just `deploy(argv).catch(helpers.showErrorAndQuit)`